src/controllers/socialAuth.js

```diff
@@ -0,0 +1,142 @@
+import axios from 'axios';
+import { config } from 'dotenv';
+import { OAuth2Client } from 'google-auth-library';
+import jwt from 'jsonwebtoken';
+
+import models from '../models';
+
+config();
+
+const {
+  FACEBOOK_APP_ID, FACEBOOK_APP_SECRET, GOOGLE_CLIENT_ID, JWT_SECRET_KEY
+} = process.env;
+const { Users } = models;
+
+/**
+ * Verify that User access token received was generated by our app and get user data from facebook
+ *
+ * @function
+ *
+ * @param {object} request.body request body
```

Comment:
Use the teams convention i.e
```suggestion
 * @param {Object} request.body request body
```